convention.md

```diff
@@ -1,53 +1,6 @@
-![Homie banner](banner.png)
+# The Homie Convention
 
-<h1 align="center">The Homie Convention</h1>
-<p align="center"><i>A lightweight MQTT convention for the IoT</i></p>
-
-<p align="center">Version: <b><!--VERSION-->3.x.x<!--VERSION--></b> • <a href="https://github.com/homieiot/convention/tags">Browse Versions…</a></p>
-
-**![Rolling Release](https://cdn2.iconfinder.com/data/icons/thesquid-ink-40-free-flat-icon-pack/64/traffic-light-20.png) The Homie Convention follows the rolling release model and is improved and released in short development cycles. Check the *releases* tab to learn about previous release versions.**
-
-Implementations of the Homie convention can be found on [this page](implementations.md).
-
-----
-
-## Table of Contents
-
-* [Motivation](#motivation)
-* [MQTT restrictions](#mqtt-restrictions)
-  * [Topic IDs](#topic-ids)
-  * [Payload](#payload)
-  * [QoS and retained messages](#qos-and-retained-messages)
-* [Topology](#topology)
-  * [Base topic](#base-topic)
-  * [Devices](#devices)
-    * [Device attributes](#device-attributes)
-    * [Device behavior](#device-behavior)
-    * [Device statistics](#device-statistics)
-  * [Nodes](#nodes)
-    * [Node attributes](#node-attributes)
-  * [Properties](#properties)
-    * [Property attributes](#property-attributes)
-  * [Arrays](#arrays)
-  * [Broadcast channel](#broadcast-channel)
-* [FAQ and Rationale](#faq)
-
-
-## Motivation
-
-The Homie convention strives to be a **communication definition on top of MQTT** between IoT devices and controlling entities.
-
-> [MQTT](http://mqtt.org) is a machine-to-machine (M2M)/"Internet of Things" connectivity protocol.
-> It was designed as an extremely lightweight publish/subscribe messaging transport.
-
-MQTT supports easy and unrestricted message-based communication.
-However, MQTT doesn't define the structure and content of these messages and their relation.
-An IoT device publishes data and provides interaction possibilities but a controlling entity will need to be specifically configured to be able to interface with the device.
-
-The Homie convention defines a **standardized way** of how IoT devices and services announce themselves and their data on the communication channel.
-The Homie convention is thereby a crucial aspect in the support of **automatic discovery, configuration and usage** of devices and services over the MQTT protocol.
-
-----
+Version: **x.x.x**
```

Comment:
I'd keep the title and the version centered. Also, I think the banner makes the document a bit prettier.

Reply:
The banner is still there in the readme, just not in the specification file itself.

The version is just a one liner, non centered like in other ietf or ieee specifications.
Personally I would like to add an auto generated date (date placeholder) even. The website mentions the release date already within the version selection box.

Reply:
Fine. Can you add a `Date: <!--DATE-->AUTO GENERATED<!--DATE-->`, and replace `Version: **x.x.x**` by `Version: **<!--VERSION-->3.x.x<!--VERSION-->**`? Thanks :)